app/views/posts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="create-post-c">
+<h2>Create a post</h2>
+
+<%= from_with model: @post, url: user_posts_path, scope: :post, html: {class: "post-form-c"} do |form| %>
```

Comment:
- [ ] Kindly change `from_with` to `form_with` as this is causing an error when trying to view posts
    
![Screenshot from 2022-06-08 14-02-28](https://user-images.githubusercontent.com/66097371/172623252-caacea38-256e-4810-aa2a-186c355ae3fa.png)